### ISSUE TITLE
MultiTerm - changes in the configuration file are ignored

### ISSUE THREAD

**Issue by diegobersanetti** [geany/geany-plugins#264]
Most of the changes made in the ~/.config/geany/plugins/multiterm/multiterm.conf are not applied; some options affected (may be more) are:
full_width_tabs
save_tabs
track_title
track_cwd

**Comment by codebrainz** [comment]
I'm not sure I implemented all the stuff in the default config file, but did you try reloading the plugin to refresh the config?

**Comment by diegobersanetti** [comment]
Yes, I tried to reload the plugin and/or closing/reopening Geany itself; I forgot to mention that I'm using Geany 1.24.1 from the Geany-Dev PPA for Ubuntu 14.04.2.

**Comment by codebrainz** [comment]
I think it's just that they aren't implemented. I should either remove them from the default config or implement them.
